janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/rest/RestElasticSearchClient.java

```diff
@@ -200,14 +205,21 @@ public void createMapping(String indexName, String typeName, Map<String,Object>
     }
 
     @Override
-    public IndexMapping getMapping(String indexName, String typeName) throws IOException{
-        Response response = performRequest(REQUEST_TYPE_GET, REQUEST_SEPARATOR + indexName + REQUEST_SEPARATOR + "_mapping" + REQUEST_SEPARATOR + typeName, null);
-        try (final InputStream inputStream = response.getEntity().getContent()) {
+    public IndexMapping getMapping(String indexName, String typeName) throws IOException {
+        InputStream inputStream = null;
+        try {
+            final Response response = performRequest(REQUEST_TYPE_GET, REQUEST_SEPARATOR + indexName + REQUEST_SEPARATOR + "_mapping" + REQUEST_SEPARATOR + typeName, null);
```

Comment:
Did you observe an exception here during your testing? The try-with-resources pattern is a little cleaner and is used elsewhere in this class.